src/modules/pushTask.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable comma-dangle */
```

Comment:
- [ ] Kindly refrain from disabling linters (unless it is permitted in the project requirements), please remove this line and all the other lines that disable linters then work on fixing the errors. They're made so we work in a specific style and we should all respect that 💯

Reply:
Okay, @ha-manel thank you for your feedback, I am getting this error because of prettier so I will have them removed.